networking/multiple_networks/primary_networks/about-primary-nwt-nad.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-primary-nwt-nad"]
+= Creating primary networks using a NetworkAttachmentDefinition
+include::_attributes/common-attributes.adoc[]
+:context: understanding-multiple-networks
+
+toc::[]
+
+The following sections explain how to create and manage additional primary networks using the `NetworkAttachmentDefinition` (NAD) resource.
+
+[id="{context}_approaches-managing-additional-network"]
+== Approaches to managing an additional network
+
+You can manage the life cycle of an additional network created by NAD with one of the following two approaches:
+
+* By modifying the Cluster Network Operator (CNO) configuration. With this method, the CNO automatically creates and manages the `NetworkAttachmentDefinition` object. In addition to managing the object lifecycle, the CNO ensures that a DHCP is available for an additional network that uses a DHCP assigned IP address.
+
+* By applying a YAML manifest. With this method, you can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
```

Comment:
Nit: Chaining is actually a different thing in CNI, without getting into [the technical details](https://github.com/containernetworking/cni/blob/main/SPEC.md#overview-1), let's avoid that term there:

> This approach allows for the chaining of CNI plugins.

to:

> This approach allows for the execution of multiple CNI plugins in order to attach to secondary networks.

or just:

> This approach allows for the execution of multiple CNI plugins

Reply:
Changed slightly.